File: 3498-reverse-degree-of-a-string/3498-reverse-degree-of-a-string.java
class Solution {
    public int reverseDegree(String s) {
        int n= s.length();
        int product = 0;
        for(int i =0; i<n; i++){
            product += (i+1)*('z' - s.charAt(i) + 1);
        }
        return product;
    }
}

File: 3498-reverse-degree-of-a-string/3498-reverse-degree-of-a-string.cpp
class Solution {
public:
    int reverseDegree(string s) {
        int pro =0, n= s.size();

        for(int i=0; i<n; i++){
            int ch = 'z' - s[i] + 1;;
            pro += (1+i) * ch;
        }

        return pro;
    }
};